ee/hogai/session_summaries/session/video_validation.py

```diff
@@ -281,13 +281,36 @@ async def _generate_updates(
             )
             return None
         updates_result = load_yaml_from_raw_llm_content(raw_content=updates_content, final_validation=True)
+        # Validate that updates_result is a list
+        if not isinstance(updates_result, list):
+            logger.error(
+                f"Invalid updates_result type for session {self.session_id}, expected list but got {type(updates_result).__name__}",
+                session_id=self.session_id,
+                signals_type="session-summaries",
+            )
+            return None
```

Comment:
Raise an exception to force the retry? If we log error anyway, and we can't do anything with the current generation, maybe it's worth to try once more?